engine/src/main/java/io/graphqlcrud/GraphQLSchemaBuilder.java

```diff
@@ -61,9 +53,19 @@ public GraphQLSchema buildSchema(Schema schema) {
         GraphQLObjectType.Builder queryTypeBuilder = GraphQLObjectType.newObject();
         queryTypeBuilder.name("QueryType");
 
+        //add filters for queries
+        builder.additionalType(Filters.pageInputBuilder().build());
```

Comment:
Awesome. I think it will be amazing to use `https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/schema/idl/SchemaPrinter.java` to print schema for us to verify this under some different model use cases (we can use snapshot testing for this or anything else to validate correctness and call it a day.

Reply:
We are printing the schema using `SchemaPrinter`, once you run you can look for the schema.

Reply:
Running this now! thank you